### PR TITLE
removing saasUserId from re

### PR DESCRIPTION
**Pull request by andrei-anfilets**

### CODE REVIEW

File: src/Softeq.NetKit.Chat.Domain/TransportModels/Response/Member/MemberSummaryResponse.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace Softeq.NetKit.Chat.Domain.TransportModels.Response.Member
     public class MemberSummaryResponse
     {
         public Guid Id { get; set; }
-
+        [JsonIgnore]
```
Comment (eugepae) [inline src/Softeq.NetKit.Chat.Domain/TransportModels/Response/Member/MemberSummaryResponse.cs]: please add TODO

File: src/Softeq.NetKit.Chat.Domain/TransportModels/Request/UserRequest.cs [hunk 2]
```diff
@@ -9,7 +11,7 @@ public UserRequest(string saasUserId)
         {
             SaasUserId = saasUserId;
         }
-
+        [JsonIgnore]
```
Comment (eugepae) [inline src/Softeq.NetKit.Chat.Domain/TransportModels/Request/UserRequest.cs]: please add TODO

Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Domain/TransportModels/Request/UserRequest.cs]: done

File: src/Softeq.NetKit.Chat.Domain/TransportModels/Response/Member/MemberSummaryResponse.cs (hunk 1 above)
Comment (andrei-anfilets) [reply src/Softeq.NetKit.Chat.Domain/TransportModels/Response/Member/MemberSummaryResponse.cs]: done

File: src/Softeq.NetKit.Chat.Domain.Services/Mappings/DomainServicesMappingProfile.cs [hunk 3]
```diff
@@ -73,7 +73,6 @@ public DomainServicesMappingProfile()
             CreateMap<Client, ClientResponse>()
```
Comment (DmitryMorozov228) [inline src/Softeq.NetKit.Chat.Domain.Services/Mappings/DomainServicesMappingProfile.cs]: Can we remove from MemberSummary model SaasUserId property?

**Pull request merged**